### ISSUE TITLE
No Cooking skill XP gained?

### ISSUE THREAD

**Issue by Dshadeslayer** [blaxxun-boop/Cooking#24]
As it says on the tin. I've been cooking a lot, and I fear Bone Appetit's goremand skill is taking precedence over your cooking skill when it comes to cooking at say the cooking station and the like.

**Comment by blaxxun-boop** [comment]
Check your log to see if cooking even started up.

**Comment by Dshadeslayer** [comment]
Will check.

**Comment by Dshadeslayer** [comment]
> Check your log to see if cooking even started up.

Extended Item Data Framework incompatibility. I saw other issues with that. That mod isn't needed for anything else, like weapon mods or such?

**Comment by blaxxun-boop** [comment]
No, the extended item data framework is deprecated for quite some time now. You can safely remove that.
